pkg/monitors/cloudfoundry/metadata.yaml

```diff
@@ -1,9 +1,37 @@
 monitors:
 - dimensions:
+    origin:
```

Comment:
I am not sure what is the main purpose of `metadata.yaml`, however I see that `genmetadata.go` probably contains code generated out of it.

I guess that we should probably either revert changes in `metadata.yaml` or regenerate `genmetadata.go`.

Reply:
Only dimensions were changed in here, and `genmetadata.go` does not use dimensions, so nothing needed to change in that.